group.hpp

```diff
@@ -77,8 +77,15 @@ class Group : public GroupInherit
     Serialize& serialize;
 
     /** @brief Custom callback when LED group is asserted
+     * Callback that holds LED group method with below mentioned parameter and
+     * return type.
```

Comment:
custom callback when LED group is asserted , it returns the asserted state based on return of the callback

Reply:
it doesn't return the asserted state. it returns if the lamp test request is handled or ignored.